airflow/jobs/job.py

```diff
@@ -265,6 +265,8 @@ def _merge_from(self, job: Job | JobPydantic | None):
     def _heartrate(job_type: str) -> float:
         if job_type == "TriggererJob":
             return conf.getfloat("triggerer", "JOB_HEARTBEAT_SEC")
+        elif job_type == "SchedulerJob":
```

Comment:
Don't we need to update it for `local task job`

Reply:
What do you mean exactly @dirrao ?

Reply:
Scheduler Job is never run as "local task job" - so I do not think so but can you point out to a concrete line of code you meant?

Reply:
Assume it was just "academic" question :)